lib/explorer/data_frame.ex

```diff
@@ -531,9 +532,9 @@ defmodule Explorer.DataFrame do
       only these columns are read into the dataframe. (default: `nil`)
 
     * `:config` - An optional config struct, normally associated with the
-      filesystem entry provided. For example, if an S3 object is being fetch,
-      then config is going to be something like `%Filesystem.S3.Config{}`.
-      Usage: `from_parquet("s3://bkt/file", config: %Filesystem.S3.Config{}`).
+      entry provided. For example, if an S3 object is being fetch,
+      then config is going to be something like `%FSS.S3.Config{}`.
+      Usage: `from_parquet("s3://bkt/file", config: %FSS.S3.Config{}`).
```

Comment:
We can say: "read can move to the moduledocs and tell people to read that instead".